tests/unit/misc/userscripts/test_qute_lastpass.py

```diff
@@ -0,0 +1,353 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2020 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for misc.userscripts.qute-lastpass."""
+
+import os
+import json
+import subprocess
+from importlib.machinery import SourceFileLoader
+from importlib.util import spec_from_loader, module_from_spec
+from unittest.mock import MagicMock, call
+
+# qute-lastpass violates naming convention and does not have .py extension
+dirname = os.path.dirname(__file__)
+script_path = os.path.join(
+    dirname, "../../../../misc/userscripts/qute-lastpass")
+spec = spec_from_loader("qute_lastpass", SourceFileLoader(
+    "qute_lastpass",
+    script_path))
+qute_lastpass = module_from_spec(spec)
+spec.loader.exec_module(qute_lastpass)
```

Comment:
Crazy stuff! I suppose using convenience methods like `importlib.import_module` won't work because they all need a Python package name?

Reply:
Yes, it's what I tried initially. This is @user202729's suggestion which works.